lib/iob/calculate.js

```diff
@@ -37,4 +58,76 @@ function iobCalc(treatment, time, dia) {
     return results;
 }
 
-exports = module.exports = iobCalc;
+
+function iobCalc(treatment, time, dia, profile) {
+
+	var curve = profile.curve;
+
+    if (curve === undefined || curve.toLowerCase() == 'bilinear') {
+    	return iobCalcBiLinear(treatment, time, dia);
+    };
+    
+    curve = curve.toLowerCase();
+    
+    if (curve != 'novorapid' && curve != 'fiasp') {
+    	console.error('Unsupported curve function: "' + curve + '". Supported curves: bilinear, novorapid and fiasp.');
+    	return;
+    }
+
+	var usePeakTime = false;
```

Comment:
It's a nitpick, but it looks like indentation is irregular here.  It's probably worth cleaning this up or using an editor to reformat the function.